okta_help.htm

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
Since this file is generated on build, should we add it to `.gitignore`?

Reply:
We want this file to be in source control and re-generated any time we make code changes to this package